client/my-sites/domain-tip/index.jsx

```diff
@@ -38,79 +37,67 @@ class DomainTip extends React.Component {
 	static propTypes = {
 		className: PropTypes.string,
 		event: PropTypes.string.isRequired,
+		hasDomainCredit: PropTypes.bool,
+		isIneligible: PropTypes.bool,
+		shouldNudgePlanUpgrade: PropTypes.bool,
 		siteId: PropTypes.number.isRequired,
-		domainsWithPlansOnly: PropTypes.bool.isRequired,
-	};
-
-	static defaultProps = {
-		quantity: noop,
+		queryObject: PropTypes.shape( {
+			query: PropTypes.string,
+			quantity: PropTypes.number,
+			vendor: PropTypes.string,
+		} ),
 	};
 
-	noticeShouldDisplay = () => {
-		// bypass some of the upgrade nudge display logic
-		return true;
-	};
-
-	domainUpgradeNudge() {
-		const { event, translate } = this.props;
-
+	renderPlanUpgradeNudge() {
 		return (
-			<UpgradeNudge
-				title={ translate( 'Get a free Custom Domain' ) }
-				message={ translate(
-					'Custom domains are free when you upgrade to a Premium or Business plan.'
-				) }
-				feature={ FEATURE_CUSTOM_DOMAIN }
-				event={ event }
-			/>
+			<div className={ classNames( 'domain-tip', this.props.className ) }>
+				<UpgradeNudge
+					event={ `domain_tip_${ this.props.event }` }
+					feature={ FEATURE_CUSTOM_DOMAIN }
+					shouldDisplay
+					message={ this.props.translate( 'Upgrade your plan to register a custom domain.' ) }
+					title={ this.props.translate( 'Get a custom domain' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Get a free Custom Domain' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domain-tip-render-logic)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
🔴